core-bundle/src/Entity/Migration.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\GeneratedValue;
+
+/**
+ * @ORM\Table(
+ *     name="tl_migration",
+ *     indexes={
+ *         @ORM\Index(name="name", columns={"name"})
+ *     }
+ * )
+ * @ORM\Entity(repositoryClass="Contao\CoreBundle\Repository\MigrationRepository")
+ */
+class Migration
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer", options={"unsigned"=true})
+     * @GeneratedValue
+     */
+    protected $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255, nullable=false)
+     */
+    protected $name;
+
+    /**
+     * @var \DateTimeInterface
+     *
+     * @ORM\Column(type="datetime", nullable=false)
```

Comment:
Should we maybe use `\DateTimeImmutable` + `datetime_immutable` ([ref](https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/types.html#datetime-immutablehere)) here?

Reply:
I'd probably also call the db field `executed_at` to be inline with Doctrine migrations.

Reply:
Well, we already have a mix here. DCA fields are camel case and in the CronJob entity we also use `lastRun` instead of `last_run` or `last_executed` for example.

Reply:
It's now renamed to `executedAt` at least, which is consistent with the other Contao entities.